src/app/ui/txLogger.js

```diff
@@ -205,7 +204,7 @@ function log (self, tx, receipt) {
   }
 }
 
-function renderKnownTransaction (self, data) {
+function renderKnownTransaction (self, data, executionContext) {
```

Comment:
Why not using `this` or `self` for `executionContext` as you're doing in the other files ?

Reply:
keeping the same behaviour for now